nipyapi/utils.py

```diff
@@ -470,7 +470,7 @@ def strip_snapshot(java_version):
     return re.sub("-SNAPSHOT", "", java_version)
 
 
```

Comment:
should this be a constant? also the doc should say the default

Reply:
This was introduced in a previous version by you, so I have no idea 😹

Reply:
Then it must be right!